src/components/BorrowerProfile/LoanUse.vue

```diff
@@ -3,8 +3,9 @@
 		{{ loanUseFiltered }}
 		<kv-text-link
 			v-if="loanId && showLearnMore"
-			:to="`/lend/${loanId}`"
+			:to="customLoanDetails? '' : `/lend/${loanId}`"
```

Comment:
Seems like we should use `null` instead of `''` (empty String) if `customLoanDetails`. Did you try this advice from Eddie? Also spacing..